packages/@lwc/wire-service/src/index.ts

```diff
@@ -90,6 +91,13 @@ export interface WireAdapterConstructor {
     new (callback: dataCallback): WireAdapter;
 }
 
+function isEmptyOrValidConfig(config: Record<string, any>): boolean {
```

Comment:
this name is confusing... is empty or valid config... I think you mean "isEmptyOrInvalid`